tests/light-data-store.rs

```diff
@@ -23,13 +23,24 @@ use lurk::{
 //       ldonCons1, ldonCons2, ldonCons3, ldonCons4]
 //     let stt := ldons.foldl (init := default) fun acc ldon =>
 //       let (_, acc) := ldon.commit acc; acc
-//     let ld : LightData := stt.exprs
-//     IO.FS.writeBinFile ⟨"foo.store"⟩ ld.toByteArray
+//     let ld : LightData := stt.store
+//     IO.FS.writeBinFile ⟨"foo.ldstore"⟩ ld.toByteArray
+// Whereas `id.ldstore` contains the data needed for typechecking the `id` function
 fn test_light_store_deserialization() {
-    let bytes = fs::read(Path::new("tests/foo.store")).unwrap();
-    let ld = LightData::de(&bytes).unwrap();
+    let directory = "tests/ldstores";
 
-    let store: LightStore<Fr> = Encodable::de(&ld).unwrap();
-
-    let _scalar_store: ScalarStore<Fr> = store.try_into().unwrap();
+    for entry in fs::read_dir(directory).expect("Failed to read directory") {
+        if let Ok(entry) = entry {
+            let path = entry.path();
+            if let Some(extension) = path.extension() {
+                if extension == "ldstore" {
+                    let file_path = path.to_str().unwrap();
+                    let file_bytes = fs::read(file_path).expect("Failed to read file");
+                    let ld = LightData::de(&file_bytes).unwrap();
+                    let ldstore: LightStore<Fr> = Encodable::de(&ld).unwrap();
+                    let _scalar_store: ScalarStore<Fr> = ldstore.try_into().unwrap();
```

Comment:
So this test is only checking successful deserialization, but nothing about the result? Fine if so, just making sure I understand the intent.